LayoutTests/TestExpectations

```diff
@@ -103,6 +103,7 @@ fast/forms/enterkeyhint-attribute-values.html [ Skip ]
 fast/scrolling/keyboard-scrolling-distance-downArrow.html [ Skip ]
 fast/scrolling/keyboard-scrolling-distance-pageDown.html [ Skip ]
 fast/scrolling/unfocusing-page-while-keyboard-scrolling.html [ Skip ]
+http/tests/notifications/permission/ [ Skip ]
```

Comment:
Please move so that it is right next to this line in this file:
`imported/w3c/web-platform-tests/permissions/ [ Skip ]`

Reply:
Fixed!